onwuvic/MyDiary
# #158909680 User can create new diary

#### What does this PR do?
User interface where a user can create a new diary

#### Description of Task to be completed?
As a User, I should be able to click on something (perhaps a button) that will navigate me to a page/interface where I can create a new entry

#### How should this be manually tested?
Given a create new entry button/link
When User click on it
Then User should see a simple and descriptive form with title, today diary form fields

#### What are the relevant pivotal tracker stories?
#158909680
